settings.gradle

```diff
@@ -32,6 +32,7 @@ gradleEnterprise {
 def subprojects = ['groovy-ant',
         'groovy-astbuilder',
         'groovy-bsf',
+        'groovy-checkers',
```

Comment:
`groovy-typecheckers` may be clearer?

Reply:
Yes, we could rename. We need something short for "type checking extensions". Perhaps 'typecheckers' is better. I was trying to avoid people second guessing should it be 'groovy-typeCheckers' or 'groovy-type-checkers' but I guess it is just something they will soon learn.

Reply:
@danielsun1106 adjusted as per your suggestion